### PR TITLE
[1/2] Stacks DLRS v0.5.0 pt1

### PR DESCRIPTION
**Pull request by DnPlas**
This is the first PR in a series of two for the DLRS v0.5.0 release.

* Updates files for pytorch-mkl image
* Updates files for pytorch-oss image
* Adds files for TVM ML Compiler

### CODE REVIEW

File: dlrs/clearlinux/ml-compiler/scripts/install_dl_frontends.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+run() {
+  echo "=============================================================="
+  printf "$(date) -- %s"
+  printf "%s\n" "$@"
+  echo "=============================================================="
+}
+
+install_dl_libs() {
+  pip --no-cache-dir install \
+    tensorflow==1.13.1 torch \
```
Comment (unrahul) [inline dlrs/clearlinux/ml-compiler/scripts/install_dl_frontends.sh]: change to 1.15

File: dlrs/clearlinux/ml-compiler/scripts/install_mkl.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
```
Comment (unrahul) [inline dlrs/clearlinux/ml-compiler/scripts/install_mkl.sh]: nit: 2019*

**Review by unrahul — CHANGES_REQUESTED** [review]
good work @DnPlas , some minor comments, please review

**Comment by DnPlas** [comment]
@unrahul just updated

**Review by rahulunair — APPROVED** [review]
+1 looks good to me, merging

**Pull request merged**